Makefile.am

```diff
@@ -99,7 +99,33 @@ po-pull:
 po-push:
 	$(MAKE) -C po $(PACKAGE_NAME).pot-update
 	rm $(srcdir)/po/{main,extra}.pot
-	$(ZANATA_CLIENT_BIN) push $(ZANATA_PUSH_ARGS)
+# This algorithm will make these steps:
+# - clone localization repository
+# - copy pot file to this repository
+# - check if pot file is changed (ignore the POT-Creation-Date otherwise it's always changed)
+# - it does not changed:
+#   - remove cloned repository
+# - it does changed:
+#   - add pot file
+#   - commit pot file
+#   - tell user to verify this file and push it in the temp dir
```

Comment:
"push it to the remote from the temp dir" ?

Reply:
Sounds much better. Fixed.